cdk/lib/__snapshots__/dotcom-components.test.ts.snap

```diff
@@ -39,93 +28,33 @@ Object {
     },
   },
   "Parameters": Object {
-    "AMI": Object {
```

Comment:
Now that our `AMI` parameter has changed to `AMIDotcomcomponents`, we need to instruct Riff-Raff to look for this, see https://github.com/guardian/support-dotcom-components/pull/518.